pkgs/top-level/release-outpaths.nix

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env nix-shell
+# When using as a callable script, passing `--argstr path some/path` overrides $PWD.
+#!nix-shell -p nix -i "nix-env -qaP --no-name --out-path --arg checkMeta true -f pkgs/top-level/release-outpaths.nix"
```

Comment:
How do you take advantage of parallelization in this code using the information from above? Or is this still to be done? I am asking because of nixpkgs-review.

Reply:
Never mind just saw: https://github.com/NixOS/nixpkgs/pull/269403